packages/salesforcedx-vscode-lwc/src/commands/forceLightningLwcStart.ts

```diff
@@ -148,6 +137,30 @@ export class ForceLightningLwcStartExecutor extends SfdxCommandletExecutor<{}> {
       this.showChannelOutput();
     });
   }
+
+  private handleErrors(
+    cancellationToken: vscode.CancellationToken,
+    serverHandler: ServerHandler,
+    exitCode: number | null | undefined
+  ) {
+    DevServerService.instance.clearServerHandler(serverHandler);
+    if (!this.serverStarted && !cancellationToken.isCancellationRequested) {
+      let message = nls.localize('force_lightning_lwc_start_failed');
+
+      // TODO proper exit codes in lwc-dev-server for address in use, auth/org error, etc.
```

Comment:
Agreed, do we have a work item for tracking that work ?

Reply:
Thanks for pointing that out, I added the last two.
W-7148356 - address already in use
W-7485707 - expired org
W-7485723 - no default org set

Reply:
As part of that can we also add a default_failure error code instead of "1".

Reply:
In this particular scenario I think "1" is appropriate. We're printing this message in the event that a server start failed, but at this point we might not know the reason why. Once we have those specific use cases I think using a different error code would be wise.

Reply:
yeah we didn't have anything created for this. my bad for leaving TODOs without work items... I try not to do that.